src/components/Loading/index.js

```diff
@@ -1,9 +1,43 @@
 import React from 'react';
+import CircleProgressBar from 'components/CircleProgressBar';
+import styles from './loading.scss';
 
-const Loading = ({ isLoading, children, onServer }) => (
-  onServer || (__CLIENT__ && isLoading) || !children
-    ? <div>Loading screen</div>
-    : children
-);
+const Loading = React.createClass({
+
+  getInitialState () {
+    return { percentage: 75 };
+  },
+
+  tick () {
+    this.setState({ percentage: this.state.percentage + 125 });
```

Comment:
@mattberridge whaddya think? the only advantage to having it in redux is time travel / replay, right? Abromov suggests [it's nbd](https://twitter.com/dan_abramov/status/725089243836588032) in this kinda thing.